src/containers/App/App.js

```diff
@@ -7,7 +7,7 @@ import Contact from 'components/contact/Contact';
 
 export function App({ t }) {
   return (
-    <div className={styles.app}>
+    <main className={styles.app}>
```

Comment:
Since we are removing all this code soon, I don't want to add more conflicts to the current PR that wipes all this code, can you rollback to `div`?

Reply:
Should I revert just the `main`? Ok, will do so tomorrow if you mean this (currently on the way home from our frontend meetup).

Reply:
Yeap, just the `main` to `div`, to avoid conflicts.

Reply:
> Yeap, just the `main` to `div`, to avoid conflicts.

If you mean the current conflicts, I can pull from upstream and rebase, so we can still use `main`. Git will then ask me to resolve it in my commit.

Reply:
https://github.com/Coding-Coach/coding-coach/pull/71 removes all this code, just leave it as it was before to avoid conflicts there.